### PR TITLE
♻️(react) use Cunningham button everywhere

### PR DESCRIPTION
**Pull request by NathanVss**
The legacy Button component was still used in some places, including
RouterButton. This commit deletes this legacy and use Cunningham
Button everywhere, with some size changes for visual coherence.

Capture d’écran 2023-12-11 à 17 10 14
Capture d’écran 2023-12-11 à 17 10 08
Capture d’écran 2023-12-11 à 17 10 02

### CODE REVIEW

File: src/frontend/js/widgets/Dashboard/components/DashboardBreadcrumbs/_styles.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@
   li {
     margin-right: 1rem;
     padding-right: 0.5rem;
+    height: 1.5rem;
```
Comment (jbpenrath) [inline src/frontend/js/widgets/Dashboard/components/DashboardBreadcrumbs/_styles.scss:22]: Sure? Why not use padding instead ?

File: src/frontend/scss/tools/_buttons.scss [hunk 2]
```diff
@@ -273,3 +273,8 @@
   margin: inherit;
   padding: inherit;
 }
+
+// Without this, bootstrap overrides Cunningham buttons style.
+a.c__button:hover {
+  text-decoration: none;
+}
```
Comment (jbpenrath) [inline src/frontend/scss/tools/_buttons.scss:280]: I think it should moved into `richie/src/frontend/scss/trumps/_bootstrap.scss`

**Review by jbpenrath — APPROVED** [review]
Great !

**Comment by jbpenrath** [comment]
Add an entry into the CHANGELOG please.

Furthermore, you should add an entry in the UPDATE.md to mention that the react component Button has been removed in favor of Cunningham Button component.

File: src/frontend/js/widgets/Dashboard/components/DashboardBreadcrumbs/_styles.scss (hunk 1 above)
Comment (NathanVss) [reply src/frontend/js/widgets/Dashboard/components/DashboardBreadcrumbs/_styles.scss:22]: Yes the icon break the height and resulting in a broken alignement :/

**Pull request merged**